src/printer.cpp

```diff
@@ -157,7 +157,7 @@ void buildMaps(const ModelPtr &model, ComponentMap &componentMap, VariableMap &v
             if (variable->equivalentVariableCount() > 0) {
                 for (size_t k = 0; k < variable->equivalentVariableCount(); ++k) {
                     VariablePtr equivalentVariable = variable->equivalentVariable(k);
-                    if (equivalentVariable->hasDirectEquivalentVariable(variable)) {
+                    if (equivalentVariable->hasEquivalentVariable(variable)) {
```

Comment:
See earlier comment.  If the default is false then this line's behaviour has changed?

Reply:
no, direct equivalence is the same as defaulting to not checking indirect equivalences. So this behaviour has not changed.